pkg/notification/manager.go

```diff
@@ -66,7 +66,17 @@ func (am *AtomicManager) Remove(id int) {
 
 // Send sends the notification to the registered handlers
 func (am *AtomicManager) Send(notification interface{}) {
-	for _, handler := range am.handlers {
+	handlers := am.copyHandlers()
+	for _, handler := range handlers {
```

Comment:
instead of copying everything at once, can we just do handler := handler inside the for loop ?

Reply:
tried didn't work.